src/components/user/LoginForm/styles.js

```diff
@@ -1,11 +1,16 @@
 import { StyleSheet } from 'react-native';
 
 const styles = StyleSheet.create({
-  button: {
-    alignSelf: 'center',
-    borderColor: 'lightblue',
-    borderRadius: 5,
-    borderWidth: 2,
+  container: {
+    width: 180,
+    marginRight: 70,
```

Comment:
can't you merge all this into margin as in a stylesheet?
Also please keep alphabetical order